conjure-java-core/src/main/java/com/palantir/conjure/java/types/BeanBuilderGenerator.java

```diff
@@ -577,7 +600,12 @@ private EnrichedField createField(FieldName fieldName, FieldDefinition field) {
                 ConjureAnnotations.withSafety(typeMapper.getClassName(type), safetyEvaluator.getUsageTimeSafety(field));
         FieldSpec.Builder spec = FieldSpec.builder(typeName, JavaNameSanitizer.sanitize(fieldName), Modifier.PRIVATE);
         if (type.accept(TypeVisitor.IS_LIST) || type.accept(TypeVisitor.IS_SET) || type.accept(TypeVisitor.IS_MAP)) {
-            spec.initializer("new $T<>()", type.accept(COLLECTION_CONCRETE_TYPE));
+            CollectionType collectionType = type.accept(COLLECTION_CONCRETE_TYPE);
+            if (collectionType.isParameterized()) {
+                spec.initializer("new $T()", collectionType.getClazz());
```

Comment:
Technically, if null collections are allowed, we should fallback to ArrayList @carterkozak. Any idea on the number of clients using this option?

Reply:
This handles that 4da29cb77ca56cb4c2ba9594516e1fbdd9454957